src/ert/config/queue_config.py

```diff
@@ -139,17 +113,130 @@ def generate_dict(option_list: List[Tuple[str, str]]) -> Dict[str, List[str]]:
             )
 
 
-def _validate_torque_options(torque_options: List[Tuple[str, str]]) -> None:
-    for option_strings in torque_options:
+memory_options: Mapping[str, List[str]] = {
+    "LSF": [],
+    "SLURM": [],
+    "TORQUE": ["MEMORY_PER_JOB"],
+    "LOCAL": [],
+}
+
```

Comment:
I can not seem to get this list of queue options that is validated to match with the list of `VALID_QUEUE_OPTIONS` found at the start of this file. Why is this?

Reply:
Union of all  these lists should `VALID_QUEUE_OPTIONS`. So the regex checks is essentially an extra verification layer on top of `VALID_QUEUE_OPTIONS`

Reply:
Hm, it's part of the docs and can be found in the code: https://github.com/equinor/ert/blob/39b7d047e5de932092251fd44a9fc212ed1460b7/src/clib/lib/job_queue/torque_driver.cpp#L210